src/components/implementation/no_interface/cbboot/quarantine.c

```diff
@@ -20,9 +20,9 @@ static struct spd_map_entry spd_map_freelist[SPD_MAP_NELEM];
 static int spd_map_freelist_index = 0;
 
 static int
-quarantine_add_to_spd_map(int o_spd, int f_spd) {
+quarantine_add_to_spd_map(int o_spd, int f_spd) {	// origin and forked
 	struct spd_map_entry *f;
-	assert(spd_map_freelist_index < SPD_MAP_NELEM);
+	assert(spd_map_freelist_index < SPD_MAP_NELEM);		
```

Comment:
You should be able to setup vim to auto-remove dead whitespace.  I'd give you the emacs code for doing that, but I have a feeling it wouldn't be useful ;-)